charts/posthog/values.yaml

```diff
@@ -1087,6 +1087,53 @@ pluginsScheduler:
   # -- Sentry endpoint to send errors to. Falls back to global sentryDSN
   sentryDSN:
 
+temporalPyWorker:
+  # -- Whether to install the PostHog Temporal Python worker stack or not.
+  enabled: false
```

Comment:
Let's keep disabled by default for now to be able to recover quickly if this were to break deploys.
A follow-up PR can switch to `true`.